### ISSUE TITLE
setting lengths

### ISSUE THREAD

**Issue by johannesbottcher** [Godi22/LaTeX-Bewerbungsvorlage#1]
You are defining new lengths, and a new interface to set them. Up till now, there is no documentation on where the lengths act or what they do. It might be easier to show the elements and use the standard way of setting lengths (at least for an experienced user). An unexperienced user will simply fail due to the huge amount of stuff. 

Sooner or later, a guy will come along asking for localization, i.e the `pagemark` in english or brazilian. If you get that done now, you will less trouble later. Since your commands are in english already ....
`scrbase` provides an interface for that. 

As already mentioned on goLaTeX.de, package `scrletter` (and the class `scrlttr2`) provide an interface to define variables. This might come in handy here. Please have a look at [BausteineBewerbung](https://github.com/johannesbottcher/bausteineBewerbung).

**Comment by Godi22** [comment]
This is correct, I define new length with a new interface (set...). But there is the Einstellungen.cfg file with an example of all lengths that can be modified.

Thanks for the info about the scrbase package. I add it to my TODO List.

Your example is nice, when I have a lot of time then I will implement it in my project. :)

**Comment by johannesbottcher** [comment]
`scrbase` is loaded by all KOMA-classes. The functionality should be availbale, since you are using `scrartcl`. By the way, have you seen [komacv](http://ctan.org/pkg/komacv)?
